pyphish/api.py

```diff
@@ -114,25 +116,38 @@ def get_all_shows(self):
         endpoint = '/shows/query/'
         self.endpoint_ = self._append_endpoint(self._base_url, endpoint)
 
-        # construct payload
-        payload = self._add_api_key_to_query_params()
-        payload['year'] = 2018
-        payload['order'] = 'DESC'
+        # initialize dataframe to hold all shows
+        all_shows = pd.DataFrame()
 
-        # request data
-        response = requests.request("GET", self.endpoint_, params=payload)
+        # query all shows by year
+        for year in range(1983, datetime.datetime.now().year+1):
```

Comment:
We should wrap up the below code into a `get_shows_by_year()`method and then just call that in `get_all_shows()`. This looks real nice though!

Reply:
good call... this is fixed with c69feca